[BrokenLinksH2]

mohitp930 opened the pull request.

opbld34 commented:
Docs Build status updates of commit _[51b61d3](https://github.com/mohitp930/onedrive-api-docs/commits/51b61d3d3809416a148f513b4c0c85a7ea4a7d9d)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/getting-started/aad-oauth.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/aad-oauth.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/aad-oauth?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-160ab145dc2ba26443ee1488e1c6da8b-c599a0a3)
[docs/rest-api/getting-started/app-registration-server.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/app-registration-server.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/app-registration-server?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-0af1aec396772909e4657e8ec892e5f6-c599a0a3)
[docs/rest-api/getting-started/authentication.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/authentication.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/authentication?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-2a88fa32436bf5ad45b2136bd75a8378-c599a0a3)
[docs/terms-of-use.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/terms-of-use.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/terms-of-use?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-23953892ae0d54db9824541cc9498d85-c599a0a3)
[docs/rest-api/concepts/migrating-from-live-sdk.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/migrating-from-live-sdk.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/migrating-from-live-sdk?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-f249aaf309890526e643caf5cdbf2eaa-c599a0a3)
[docs/rest-api/concepts/permissions_reference.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/permissions_reference.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/permissions_reference?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-623c6cb58764f6fdff5dcf4c1dd419b4-c599a0a3)

<a id="160ab145dc2ba26443ee1488e1c6da8b-c599a0a3"></a>
### [docs/rest-api/getting-started/aad-oauth.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/aad-oauth.md)
 - **Line 39, Column 26**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/discover-service-endpoints' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 62, Column 5**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/common-app-authentication-tasks' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 247, Column 41**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645538.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 281, Column 42**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645540.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 291, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://azure.microsoft.com/en-us/documentation/articles/web-sites-authentication-authorization/' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 293, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/common-app-authentication-tasks' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 294, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645545.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 295, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645542.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="0af1aec396772909e4657e8ec892e5f6-c599a0a3"></a>
### [docs/rest-api/getting-started/app-registration-server.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/app-registration-server.md)
 - **Line 83, Column 56**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/setup-development-environment#bk_CreateAzureSubscription' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 91, Column 36**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/add-common-consent-manually' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="2a88fa32436bf5ad45b2136bd75a8378-c599a0a3"></a>
### [docs/rest-api/getting-started/authentication.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/authentication.md)
 - **Line 38, Column 53**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://azure.microsoft.com/en-us/documentation/articles/active-directory-aadconnect/' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 68, Column 49**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://technet.microsoft.com/en-us/library/mt728971.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="23953892ae0d54db9824541cc9498d85-c599a0a3"></a>
### [docs/terms-of-use.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/terms-of-use.md)
 - **Line 20, Column 10**: **[Warning: insecure-link - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/insecure-link?branch=main)]** `````Link 'http://go.microsoft.com/fwlink/p/?LinkId=137876' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 21, Column 8**: **[Warning: insecure-link - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/insecure-link?branch=main)]** `````Link 'http://go.microsoft.com/fwlink/p/?LinkId=74170' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````
 - **Line 11, Column 176**: **[Suggestion: docs-link-absolute - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/legal/microsoft-apis/terms-of-use' will be broken in isolated environments. Replace with a relative link.`````

<a id="f249aaf309890526e643caf5cdbf2eaa-c599a0a3"></a>
### [docs/rest-api/concepts/migrating-from-live-sdk.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/migrating-from-live-sdk.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="623c6cb58764f6fdff5dcf4c1dd419b4-c599a0a3"></a>
### [docs/rest-api/concepts/permissions_reference.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/permissions_reference.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/1b9326b4-eb89-99a2-12be-09cf82b65094/202207252147151068-1611/BuildReport?accessString=ae7bc921c3199f03478e6b260313100670654af4301b0479ac58d7e6002c3b6a).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/1b9326b4-eb89-99a2-12be-09cf82b65094?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld30 commented:
Docs Build status updates of commit _[f82922c](https://github.com/mohitp930/onedrive-api-docs/commits/f82922c9147c905684cdebf82b578e4e209c2acc)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/getting-started/aad-oauth.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/aad-oauth.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/aad-oauth?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-160ab145dc2ba26443ee1488e1c6da8b-eb2007de)
[docs/rest-api/getting-started/app-registration-server.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/app-registration-server.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/app-registration-server?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-0af1aec396772909e4657e8ec892e5f6-eb2007de)
[docs/rest-api/getting-started/authentication.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/authentication.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/authentication?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-2a88fa32436bf5ad45b2136bd75a8378-eb2007de)
[docs/terms-of-use.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/terms-of-use.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/terms-of-use?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-23953892ae0d54db9824541cc9498d85-eb2007de)
[docs/rest-api/concepts/migrating-from-live-sdk.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/migrating-from-live-sdk.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/migrating-from-live-sdk?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-f249aaf309890526e643caf5cdbf2eaa-eb2007de)
[docs/rest-api/concepts/permissions_reference.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/permissions_reference.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/permissions_reference?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-623c6cb58764f6fdff5dcf4c1dd419b4-eb2007de)

<a id="160ab145dc2ba26443ee1488e1c6da8b-eb2007de"></a>
### [docs/rest-api/getting-started/aad-oauth.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/aad-oauth.md)
 - **Line 39, Column 26**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/discover-service-endpoints' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 62, Column 5**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/common-app-authentication-tasks' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 247, Column 41**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645538.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 281, Column 42**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645540.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 291, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://azure.microsoft.com/en-us/documentation/articles/web-sites-authentication-authorization/' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 293, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/common-app-authentication-tasks' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 294, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645545.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 295, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645542.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="0af1aec396772909e4657e8ec892e5f6-eb2007de"></a>
### [docs/rest-api/getting-started/app-registration-server.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/app-registration-server.md)
 - **Line 83, Column 56**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/setup-development-environment#bk_CreateAzureSubscription' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 91, Column 36**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/add-common-consent-manually' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="2a88fa32436bf5ad45b2136bd75a8378-eb2007de"></a>
### [docs/rest-api/getting-started/authentication.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/authentication.md)
 - **Line 38, Column 53**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://azure.microsoft.com/en-us/documentation/articles/active-directory-aadconnect/' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 68, Column 49**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://technet.microsoft.com/en-us/library/mt728971.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="23953892ae0d54db9824541cc9498d85-eb2007de"></a>
### [docs/terms-of-use.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/terms-of-use.md)
 - **Line 20, Column 10**: **[Warning: insecure-link - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/insecure-link?branch=main)]** `````Link 'http://go.microsoft.com/fwlink/p/?LinkId=137876' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 21, Column 8**: **[Warning: insecure-link - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/insecure-link?branch=main)]** `````Link 'http://go.microsoft.com/fwlink/p/?LinkId=74170' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````
 - **Line 11, Column 176**: **[Suggestion: docs-link-absolute - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/legal/microsoft-apis/terms-of-use' will be broken in isolated environments. Replace with a relative link.`````

<a id="f249aaf309890526e643caf5cdbf2eaa-eb2007de"></a>
### [docs/rest-api/concepts/migrating-from-live-sdk.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/migrating-from-live-sdk.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="623c6cb58764f6fdff5dcf4c1dd419b4-eb2007de"></a>
### [docs/rest-api/concepts/permissions_reference.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/permissions_reference.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/1b9326b4-eb89-99a2-12be-09cf82b65094/202207261406523478-1611/BuildReport?accessString=3723384d089420f99856900e05a550ebc2e2621e89315cc06bea0be897ddbafb).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/1b9326b4-eb89-99a2-12be-09cf82b65094?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

mohitp930 commented:
Global effort to fix broken links

@eaverett4

[Note] There are quite a few validation issues associated with these files. My work is tightly scoped to just broken links, and a separate validation team should be addressing these issues in the future. Let me know how you'd like to proceed.

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. Thanks!

opbld32 commented:
Docs Build status updates of commit _[f82922c](https://github.com/mohitp930/onedrive-api-docs/commits/f82922c9147c905684cdebf82b578e4e209c2acc)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/getting-started/aad-oauth.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/aad-oauth.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/aad-oauth?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-160ab145dc2ba26443ee1488e1c6da8b-000cb5ce)
[docs/rest-api/getting-started/app-registration-server.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/app-registration-server.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/app-registration-server?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-0af1aec396772909e4657e8ec892e5f6-000cb5ce)
[docs/rest-api/getting-started/authentication.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/authentication.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/authentication?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-2a88fa32436bf5ad45b2136bd75a8378-000cb5ce)
[docs/terms-of-use.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/terms-of-use.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/terms-of-use?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-23953892ae0d54db9824541cc9498d85-000cb5ce)
[docs/rest-api/concepts/migrating-from-live-sdk.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/migrating-from-live-sdk.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/migrating-from-live-sdk?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-f249aaf309890526e643caf5cdbf2eaa-000cb5ce)
[docs/rest-api/concepts/permissions_reference.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/permissions_reference.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/permissions_reference?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-623c6cb58764f6fdff5dcf4c1dd419b4-000cb5ce)

<a id="160ab145dc2ba26443ee1488e1c6da8b-000cb5ce"></a>
### [docs/rest-api/getting-started/aad-oauth.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/aad-oauth.md)
 - **Line 39, Column 26**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/discover-service-endpoints' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 62, Column 5**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/common-app-authentication-tasks' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 247, Column 41**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645538.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 281, Column 42**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645540.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 291, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://azure.microsoft.com/en-us/documentation/articles/web-sites-authentication-authorization/' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 293, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/common-app-authentication-tasks' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 294, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645545.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 295, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645542.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="0af1aec396772909e4657e8ec892e5f6-000cb5ce"></a>
### [docs/rest-api/getting-started/app-registration-server.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/app-registration-server.md)
 - **Line 83, Column 56**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/setup-development-environment#bk_CreateAzureSubscription' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 91, Column 36**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/add-common-consent-manually' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="2a88fa32436bf5ad45b2136bd75a8378-000cb5ce"></a>
### [docs/rest-api/getting-started/authentication.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/authentication.md)
 - **Line 38, Column 53**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://azure.microsoft.com/en-us/documentation/articles/active-directory-aadconnect/' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 68, Column 49**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://technet.microsoft.com/en-us/library/mt728971.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="23953892ae0d54db9824541cc9498d85-000cb5ce"></a>
### [docs/terms-of-use.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/terms-of-use.md)
 - **Line 20, Column 10**: **[Warning: insecure-link - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/insecure-link?branch=main)]** `````Link 'http://go.microsoft.com/fwlink/p/?LinkId=137876' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 21, Column 8**: **[Warning: insecure-link - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/insecure-link?branch=main)]** `````Link 'http://go.microsoft.com/fwlink/p/?LinkId=74170' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````
 - **Line 11, Column 176**: **[Suggestion: docs-link-absolute - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/legal/microsoft-apis/terms-of-use' will be broken in isolated environments. Replace with a relative link.`````

<a id="f249aaf309890526e643caf5cdbf2eaa-000cb5ce"></a>
### [docs/rest-api/concepts/migrating-from-live-sdk.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/migrating-from-live-sdk.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="623c6cb58764f6fdff5dcf4c1dd419b4-000cb5ce"></a>
### [docs/rest-api/concepts/permissions_reference.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/permissions_reference.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/1b9326b4-eb89-99a2-12be-09cf82b65094/202208021440581114-1611/BuildReport?accessString=88446e3467dc05f13d872a3b7367b721bb5cac5315001a05c11b49cd997402b7).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/1b9326b4-eb89-99a2-12be-09cf82b65094?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[bad9f44](https://github.com/mohitp930/onedrive-api-docs/commits/bad9f4454cba751d26f5c076ad47b84985c12ea1)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/getting-started/aad-oauth.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/aad-oauth.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/aad-oauth?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-160ab145dc2ba26443ee1488e1c6da8b-7e6c2836)
[docs/rest-api/getting-started/app-registration-server.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/app-registration-server.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/app-registration-server?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-0af1aec396772909e4657e8ec892e5f6-7e6c2836)
[docs/rest-api/getting-started/authentication.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/authentication.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/getting-started/authentication?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-2a88fa32436bf5ad45b2136bd75a8378-7e6c2836)
[docs/terms-of-use.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/terms-of-use.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/terms-of-use?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-23953892ae0d54db9824541cc9498d85-7e6c2836)
[docs/rest-api/concepts/migrating-from-live-sdk.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/migrating-from-live-sdk.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/migrating-from-live-sdk?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-f249aaf309890526e643caf5cdbf2eaa-7e6c2836)
[docs/rest-api/concepts/permissions_reference.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/permissions_reference.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/permissions_reference?view=odsp-graph-online&branch=pr-en-us-1611) (>= odsp-graph-online) | [Details](#user-content-623c6cb58764f6fdff5dcf4c1dd419b4-7e6c2836)

<a id="160ab145dc2ba26443ee1488e1c6da8b-7e6c2836"></a>
### [docs/rest-api/getting-started/aad-oauth.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/aad-oauth.md)
 - **Line 39, Column 26**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/discover-service-endpoints' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 62, Column 5**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/common-app-authentication-tasks' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 247, Column 41**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645538.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 281, Column 42**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645540.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 291, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://azure.microsoft.com/en-us/documentation/articles/web-sites-authentication-authorization/' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 293, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/common-app-authentication-tasks' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 294, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645545.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 295, Column 3**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/library/azure/dn645542.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="0af1aec396772909e4657e8ec892e5f6-7e6c2836"></a>
### [docs/rest-api/getting-started/app-registration-server.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/app-registration-server.md)
 - **Line 83, Column 56**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/setup-development-environment#bk_CreateAzureSubscription' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 91, Column 36**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://msdn.microsoft.com/en-us/office/office365/howto/add-common-consent-manually' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="2a88fa32436bf5ad45b2136bd75a8378-7e6c2836"></a>
### [docs/rest-api/getting-started/authentication.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/getting-started/authentication.md)
 - **Line 38, Column 53**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://azure.microsoft.com/en-us/documentation/articles/active-directory-aadconnect/' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 68, Column 49**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://technet.microsoft.com/en-us/library/mt728971.aspx' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="23953892ae0d54db9824541cc9498d85-7e6c2836"></a>
### [docs/terms-of-use.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/terms-of-use.md)
 - **Line 20, Column 10**: **[Warning: insecure-link - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/insecure-link?branch=main)]** `````Link 'http://go.microsoft.com/fwlink/p/?LinkId=137876' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 21, Column 8**: **[Warning: insecure-link - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/insecure-link?branch=main)]** `````Link 'http://go.microsoft.com/fwlink/p/?LinkId=74170' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````
 - **Line 11, Column 176**: **[Suggestion: docs-link-absolute - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/legal/microsoft-apis/terms-of-use' will be broken in isolated environments. Replace with a relative link.`````

<a id="f249aaf309890526e643caf5cdbf2eaa-7e6c2836"></a>
### [docs/rest-api/concepts/migrating-from-live-sdk.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/migrating-from-live-sdk.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="623c6cb58764f6fdff5dcf4c1dd419b4-7e6c2836"></a>
### [docs/rest-api/concepts/permissions_reference.md](https://github.com/mohitp930/onedrive-api-docs/blob/mp7252022-broken-links/docs/rest-api/concepts/permissions_reference.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/1b9326b4-eb89-99a2-12be-09cf82b65094/202208051421339948-1611/BuildReport?accessString=de1dc4acb371a9dfc597c3ea018fc37b0506e98201b2fa05bb17e48f0a8c1e7e).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/1b9326b4-eb89-99a2-12be-09cf82b65094?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

tfosmark commented:
@mohitp930 Please rebase this PR against the main branch (instead of live), and then fix the warnings and the absolute link suggestion. Thanks!
